src/utils/httpClient.ts

```diff
@@ -39,24 +40,30 @@ export const requestHandler = async <TResponse, TPayload = undefined>(
         break;
     }
 
+    const status = response.data.status;
+
+    if (status !== 'SUCCESS') {
+      throw new InternalAPIError(response.data.data as string, status);
```

Comment:
해당 부분이 
```suggestion
if (status !== 'SUCCESS') {
  return Promise.reject(new InternalAPIError(response.data.data as string, status));
}
```
로 lint 에러를 우회할 수 있다고는 하는데, exception 의 처리 방식을 달리해봐도 될 거 같아요!

Reply:
일단 제안해주신 우회 방법으로 처리해두었어요! 
찾아보니까 lint에서는 try문에서 에러를 던지지말고 바로 처리하는 걸 권장하기 때문에 발생하는 에러더라구요
하지만 try문에서는 useDialogActions 과 같은 훅을 사용하지 못해서 에러 처리가 분산될 것 같아 제안 주신 방법으로 진행했습니다! [49c54fb](https://github.com/JECT-Study/JECT-Official-WebSite-Client/pull/143/commits/49c54fb7d227addcf66d5a36774f3d616ddb4f95)